routes/user.js

```diff
@@ -2,6 +2,7 @@ var express = require('express');
 var router = express.Router();
 var passport = require('passport');
 var User = require('../models/user');
+var Playlist = require('../models/playlist')
```

Comment:
add the semicolon to the end. Let's just keep to that standard for all javascript statements.